.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "\U0001F41B Bug Report"
+about: "Please report issues related to the TypeScript and HTML API of Vaadin components here."
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue risks taking an infinite amount of time to triage. This info allows the maintainers team to diagnose and fix your issue as quickly as possible.
+
+  Before opening a new issue, please search existing issues: https://github.com/vaadin/web-components/issues.
+-->
+
+### Description
+
+Describe the issue that you're seeing.
+
+<!-- Example: The `vaadin-grid` element scrolls when applying filter. -->
+
+### Expected outcome
+
+What should happen?
+
+<!-- Example: No scrolling should happen. -->
+
+### Actual outcome
+
+What happened.
+
+<!-- Example: Grid content occasionally gets scrolled. -->
+
+### Live Demo
+
+A screenshot or video with issue reproduced.
+
+### Minimal reproducible example
+
+**Please** provide a code snippet with issue reproduction, this makes your issue _much_ easier to diagnose.
+
+<!--
+```
+<vaadin-grid</vaadin-grid>
+```
+```
+const grid = document.querySelector('vaadin-grid');
+grid.items = items;
+```
+-->
+
+### Steps to reproduce
+
+Clear steps describing how to reproduce the issue.
+
+<!-- Example
+1. Put a `vaadin-grid` element in the page.
+2. Add `vaadin-grid-filter` element to the column header template.
+3. Open the page in a web browser.
+4. Start typing in the filter input.
+-->
+
+### Environment
+
+- Version of the webcomponent / platform.
```

Comment:
This might not be clear enough. For the most part we want to support Flow / Fusion users, so they should only need to specify the Vaadin platform version. For standalone web-component usage we would indeed need the web component version, but that is also aligned with the platform since 20. So maybe just mention the platform version here?

Reply:
I don't have strong preferences on this one but would like to show that users can specify webcomponent versions for the components used in 14.x where the versions are not aligned.

Reply:
OK, one small improvement I would suggest is to use the term `Vaadin platform`

Reply:
Sounds good! Updated.